src/common/components/cards/rule-resources.tsx

```diff
@@ -17,6 +18,10 @@ export type RuleResourcesProps = {
 };
 
 export const RuleResources = NamedFC<RuleResourcesProps>('RuleResources', ({ deps, rule }) => {
+    if (rule.url == null || isEmpty(rule.guidance)) {
```

Comment:
Is this really intended to be `||` rather than `&&`? I'd have expected that we want to show the resources in the event that we have either of them

Reply:
Make sense